assets/base.css

```diff
@@ -1191,37 +1191,71 @@ deferred-media {
   position: relative;
 }
 
+.share-button__button {
+  font-size: 1.4rem;
+  display: flex;
+  align-items: baseline;
+  color: rgba(var(--color-link),var(--alpha-link));
+  margin-left: 0;
+  padding-left: 0;
+
+}
+
+.share-button__button, .share-button__fallback button {
+  cursor: pointer;
+  background: var(--color-background);
+  border: none;
+}
+
 .share-button__button .icon-share {
   margin-right: 1rem;
 }
 
 .share-button__fallback {
-  border: 0.1rem solid rgba(var(--color-foreground), 0.1);
   background: rgb(var(--color-background));
-  padding: 2rem;
+  display: flex;
   position: absolute;
-  top: 4rem;
+  top: 3rem;
   left: -0.1rem;
   z-index: 3;
   width: 100%;
-  display: flex;
-  flex-direction: column;
+  box-shadow: 0 0 0 .1rem rgba(var(--color-foreground),.55);
 }
 
-.share-button__fallback .button {
-  margin-top: 1rem;
+.share-button__fallback button {
+  width: 4.4rem;
+  height: 4.4rem;
+  padding: 0;
+  flex-shrink: 0;
+}
+
+.share-button__close:not(.hidden) + .share-button__copy {
+  display: none;
 }
 
-.share-button__fallback .icon-clipboard {
-  height: 1.4rem;
-  width: 1.8rem;
+.share-button__fallback .field__input {
+  box-shadow: none;
+  text-overflow: ellipsis;
+}
+
+.share-button__fallback .icon {
+  width: 1.5rem;
+  height: 1.5rem;
 }
 
```

Comment:
Success message feels a bit tight on mobile:

https://screenshot.click/09-54-5yjwc-zbis3.png

Reply:
+1. When the text goes onto two lines, the entire box should get larger so the padding around doesn't change.

Reply:
Does this looks good to you Olivia ? [video](https://screenshot.click/21-08-e6wqd-b9a4e.mp4)
But I should probably keep the close button vertically centered.

Reply:
This looks good!